src/pages/ErrorPage/GenericErrorPage.js

```diff
@@ -14,6 +14,7 @@ import styles from '../../styles/styles';
 import ErrorBodyText from './ErrorBodyText';
 import TextLink from '../../components/TextLink';
 import CONST from '../../CONST';
+import colors from '../../styles/colors';
```

Comment:
We should not import colors directly, see https://github.com/Expensify/App/blob/b89f5d8ed8dfa7c22c02d7c4e1acfc2a33a2308d/src/styles/colors.js#L2

Reply:
Good catch @luacmartins

Reply:
Done